config/webpack/environment.js

```diff
@@ -1,3 +1,19 @@
 const { environment } = require('@rails/webpacker')
 
-module.exports = environment.toWebpackConfig()
+// Add global webpack configs here
+
+const babelLoader = environment.loaders.get('babel')
+babelLoader.test = /\.jsx?|.es6?|.spec.js?$/
+
+// css/scss/sass loaders are included by default, but we redefine them for some reason. TODO: Ask Didier.
```

Comment:
you are checking master documentation. Probably back in a day I was hitting something similar to https://github.com/rails/webpacker/issues/756#issuecomment-327148547

Reply:
It might, I'm not familiar with the current settings, or latest API, so make the change and make sure the styles are loaded correctly both in views and modules.

Reply:
They do, everything looks the same although Percy is complaining.